### ISSUE TITLE
Problem to install

### ISSUE THREAD

**Issue by vallettea** [edouard-lopez/ECV-IJBA#1]
Hello Edouard,

I'am beginner at frontend so I might I've installed something in a wrong way but when I try to follow the readme i get:

```
⇒  npm install
npm WARN package.json itineraire-dechets@0.0.0 No repository field.
npm WARN prefer global bower@1.3.4 should be installed with -g
npm WARN prefer global grunt-cli@0.1.13 should be installed with -g
npm WARN prefer global yo@1.1.2 should be installed with -g

> yo@1.1.2 postinstall /Users/vallette/ANTS/ecv-ijba/node_modules/yo
> node ./scripts/doctor

[Yeoman Doctor] Everything looks alright!

vallette@Alexandres-MacBook-Pro:~/ANTS/ecv-ijba|master
⇒  bower install
vallette@Alexandres-MacBook-Pro:~/ANTS/ecv-ijba|master
⇒  gulp serve
[gulp] Using file /Users/vallette/ANTS/ecv-ijba/gulpfile.js
[gulp] Working directory changed to /Users/vallette/ANTS/ecv-ijba

module.js:340
    throw err;
          ^
Error: Cannot find module 'through2'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/Users/vallette/ANTS/ecv-ijba/node_modules/gulp-useref/index.js:2:15)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
vallette@Alexandres-MacBook-Pro:~/ANTS/ecv-ijba|master
```

**Comment by edouard-lopez** [comment]
Hi Alex,
sorry for the delay I've been traveling. 
My _readme_ assume you already have a working `NPM` environment with some essential packages ([`bower`](http://bower.io/), [`grunt`](http://gruntjs.com/)/`gulp`, ~~`yeoman`~~[`yo`](http://yeoman.io/), etc.). 
The `npm install` have installed them locally it seems (the first warnings) but they need to be installed globally

So remove the `node_modules` directory from the project (or restart from scratch) and try:

```
npm cache clean; bower cache clean
npm install -g yo bower grunt-cli gulp topojson 
```

Let me know if you still have problems,
cheers.

**Comment by vallettea** [comment]
fixed by cbcf1886d172b8d1a2ec8ff74ba8ebaf4cfd1ddf
